contents/handbook/small-teams/marketing/objectives.mdx

```diff
@@ -1,18 +1,36 @@
 * **Objective**:
-    * Expand PostHog’s top of funnel awareness across all our core _individual_ products - product analytics, session recording, feature flags, and A/B testing.
+    * Increase word-of-mouth growth by:
+      1. being an authority and vital resource on building successful products.
+      2. creating a world-class community experience.
 
-* **Key Results:**
-    * Increase [overall weekly website traffic](https://app.posthog.com/insights/hg6kfoIG) to 27k unique visitors per week (filtering out viral spikes).
-    * Increase [‘first user in org’ signups](https://app.posthog.com/insights/EwWz7QuV) to 600 per week without decreasing the percentage that are ICPs (20-25%).
-    * Make Tutorials our [3rd most visited section](https://app.posthog.com/insights/n8b9SHgd) (after Docs and Blogs)
-    * Increase number of [subscribers](https://newsletter.posthog.com/publish/subscribers) to Product for Engineers to 1,500
-    * Get [200 orgs](https://docs.google.com/spreadsheets/d/1ryDGzXcG0OSH-GoX-zlI2CKwNu-6Zivi7EM32WLdids/edit#gid=0) to sign up to the startup deal
-    * Increase [28 day YouTube watch time](https://studio.youtube.com/channel/UCn4mJ4kK5KVSvozJre645LA?c=UCn4mJ4kK5KVSvozJre645LA) to 227 hours
+* **Goals:**
+    * Nail Twitter and content distribution (**Owner:** TBD, maybe Charles?)
+    * Nail internal content discovery and engagement (**Owner:** Ian with Website & Docs)
+    * Make PostHog for Startups the default choice for technical founders (**Owner:** Joe)
+    * Scale the newsletter to thousands of engaged subscribers (**Owner:** Andy)
+    * Create and execute a community plan (**Owner:** Lior)
+    * Ship printed PMF guide with a distribution plan (**Owner:** Lottie)
 
 * **Rationale:**
-    * We frequently get brand mentions where people are saying ‘Amplitude, Mixpanel or PostHog’. We want them to also say things like ‘LaunchDarkly or PostHog’, ‘FullStory or PostHog’, and 'Optimizely or PostHog'.
-    * While SEO remains very important to us, we want to diversify our audience slightly so we are not so dependent on it as a single channel as a defensive move, especially if there are AI-related changes to search in future. Building things like our newsletter, YouTube channel, and our startup community will help us do this. 
-    * While tutorials often do not drive top of funnel awareness for people who have never heard of PostHog, they will help drive awareness of _other products_ once users are signed up to PostHog. 
+    * Previously we've set metric-centric goals. This made sense when BAU work drove growth easily, but we feel marketing and content needs to become more mission driven to drive long-term bottom-up growth. We still monitor several metrics, but we treat them as indicators of progress rather than discrete goals.
```

Comment:
```suggestion
    * Previously we've set metric-centric goals. This made sense when business-as-usual work drove growth easily, but we feel marketing and content needs to become more mission driven to drive long-term bottom-up growth. We still monitor several metrics, but we treat them as indicators of progress rather than discrete goals.
```
I did not know what this meant